packages/core-data/src/hooks/test/use-entity-record.js

```diff
@@ -113,4 +114,41 @@ describe( 'useEntityRecord', () => {
 		expect( widget.editedRecord ).toEqual( { hello: 'foo', id: 1 } );
 		expect( widget.edits ).toEqual( { hello: 'foo' } );
 	} );
+
+	it( 'does not resolve entity record when disabled via options', async () => {
+		// Provide response
+		triggerFetch.mockImplementation( () => TEST_RECORD );
+
+		let data;
+		const TestComponent = () => {
+			data = useEntityRecord( 'root', 'widget', 2, {
+				options: { enabled: false },
```

Comment:
This should just be:

```suggestions
{ enabled: false },
```

Passing any object still works because we're only defaulting `enabled` to `true` if `options` is `undefined`. Otherwise, `options.enabled` will always be falsy (`undefined`) unless explicitly set to `true`. I think this is another bug we should fix.